src/nautilus_librarian/typer/commands/workflows/actions/auto_commit_base_images.py

```diff
@@ -139,7 +142,9 @@ def process_added_base_images(
 
         guard_that_base_image_exists(base_img_absolute_path)
 
-        commit_new_base_image(git_repo_dir, base_img_relative_path, gnupghome, git_user)
+        commit_new_modified_base_image(
```

Comment:
`commit_modified_base_image` (without `new`)?

Reply:
This naming is because the method is used both for new AND modified
Maybe I should have used "new_and_modified", but sometimes I omit particles for shortening when I see that doesn't sacrifice legibility. Maybe in this case indeed lacks legibility.